src/pages/dataTableUtilities/actions/cellActions.js

```diff
@@ -36,6 +36,53 @@ export const refreshIndicatorRow = route => ({
   payload: { route },
 });
 
+export const editBatchDoses = (newValue, rowKey, route) => (dispatch, getState) => {
+  const { data, keyExtractor } = selectPageState(getState());
+
+  const objectToEdit = data.find(row => keyExtractor(row) === rowKey);
+
+  if (objectToEdit) {
+    UIDatabase.write(() => objectToEdit.setDoses(UIDatabase, parsePositiveInteger(newValue)));
+    dispatch(refreshRow(rowKey, route));
+  }
+};
+
+export const editBatchVvmStatus = (vvmStatus, objectType, route) => (dispatch, getState) => {
+  const { modalValue, keyExtractor } = selectPageState(getState());
+
+  UIDatabase.write(() =>
+    UIDatabase.update(objectType, { ...modalValue, vaccineVialMonitorStatus: vvmStatus })
+  );
+
+  reduxBatch(() => {
```

Comment:
I get the idea of the `reduxBatch` rename when dealing with vaccine batches, but something about it feels wrong to me! I tend to rename local variables rather than imported ones, as it maintains consistency across files (e.g. if I want to do a global search for all the times `batch` is used. I think if we rename like this, we should do this consistently across all files?

Reply:
actually it was already renamed ages ago - theres a method which has a param `batch` cause it can be a transactionbatch or stocktake batch and I DONT KNOW another name :(